worlds/aquaria/Regions.py

```diff
@@ -16,7 +16,7 @@
 
 def _has_hot_soup(state:CollectionState, player: int) -> bool:
```

Comment:
There's a bunch of missing spaces before the word "CollectionState" in the methods of this file.
You could run a reformat file and get it all cleaned up at once

Reply:
I will look at that. Thank you.